execution_environments/README.md

```diff
@@ -6,9 +6,14 @@ Currently these execution environment images are created manually using the `bui
 
 ## Building the execution environment images
 
-1. `podman login registry.redhat.io` in order to pull the base EE images
-2. `./build.sh` to build the EE images and add them to your local podman image cache
+1. Download the [openshift-clients rpm](https://access.redhat.com/downloads/content/rhel---9/x86_64/20821/openshift-clients/4.16.0-202408021139.p0.ge8fb3c0.assembly.stream.el9/x86_64/fd431d51/package)
+2. Overwrite existing link to git-lfs by copying downloaded RPM to `<repo-root>/execution-environments/openshift-clients-4.16.0-202408021139.p0.ge8fb3c0.assembly.stream.el9.x86_64.rpm`
```

Comment:
can we drop these two bullets for now?  i think git-lfs is the right path forward so the documentation should be updated to take that into account.

Reply:
removed